static/src/javascripts/projects/commercial/modules/dfp/prepare-prebid.js

```diff
@@ -10,7 +10,8 @@ const isGoogleWebPreview: () => boolean = () =>
     !!(
         navigator &&
         navigator.userAgent &&
-        navigator.userAgent.indexOf('Google Web Preview') > -1
+        (navigator.userAgent.indexOf('Google Web Preview') > -1 ||
+            navigator.userAgent.indexOf('googleweblight') > -1)
```

Comment:
Could you reduce the duplicate code a bit?

Reply:
@kelvin-chappell I think the duplication is fine, otherwise you just have a solution to not have duplication.

For example

```javascript
const isGoogleWebPreview: () => boolean = () => {
    const userAgentContains = (userAgent: string): boolean =>
        navigator.userAgent.indexOf(userAgent) > 1;
    return !!(
        navigator &&
        navigator.userAgent &&
        (userAgentContains('Google Web Preview') ||
            userAgentContains('googleweblight'))
    );
};
```